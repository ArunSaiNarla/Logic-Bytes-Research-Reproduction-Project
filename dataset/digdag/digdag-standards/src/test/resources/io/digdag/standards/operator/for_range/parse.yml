_command: '{"from":0,"to":10,"slices":3}'
_do:
  echo>: do
