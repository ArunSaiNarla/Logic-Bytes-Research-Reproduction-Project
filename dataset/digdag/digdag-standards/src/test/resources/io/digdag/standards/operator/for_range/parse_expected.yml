+range-from=0&to=4:
  echo>: do
  _export:
    range: {from: 0, to: 4, index: 0}
+range-from=4&to=8:
  echo>: do
  _export:
    range: {from: 4, to: 8, index: 1}
+range-from=8&to=10:
  echo>: do
  _export:
    range: {from: 8, to: 10, index: 2}
